api/config.yaml

```diff
@@ -140,19 +140,17 @@ safety:
     git clone -b %GIT_BRANCH% --single-branch %GIT_REPO% code --quiet 2> /tmp/errorGitCloneSafety
     if [ $? -eq 0 ]; then
       cd code
-      if [ $(cat requirements.txt | grep '=' | grep -v '#' 1> safety_huskyci_analysis_requirements.txt 2> /tmp/errorFindRequirements) ]; then
```

Comment:
I think that maybe keeping this `if` might be useful to avoid commented out librarys from being analyzed by huskyCI. 

What do you think, @rafaveira3?